lib/tasks/seed_users_with_profiles.rake

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
```

Comment:
Do you think we should create data that should not be updated to ensure/check that we are not updating establishing, pending, and passed enrollments and/or those enrollments that don't have a timestamp for the profile? What are your thoughts on this? Overkill?

Reply:
I like this idea if it's not too much trouble!